index.js

```diff
@@ -4,8 +4,8 @@ const https = require('https')
 const chalk = require('chalk')
 const argv = require('yargs')
   .command('sunsign', "sunsign to get today's horoscope for")
-  .argv.help('help')
   .help('help')
+  .argv
```

Comment:
_(idle question, not a change request for this PR)_ If the code chained after `const argv = require('yargs')` can potentially cause user facing errors, then it seems like it would be worthwhile to put it inside of a function call? So you can do some error checking, if nothing else.

Reply:
if you're talking about the chain itself, AFAIK there's no errors that _should_ potentially occur there - I may be misunderstanding, though. If you're talking about the code after the chaining, yes absolutely.